packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -115,23 +125,63 @@ export class BacktraceDatabase implements BacktraceModule {
     public add(
         backtraceData: BacktraceData,
         attachments: BacktraceAttachment<unknown>[],
-    ): BacktraceDatabaseRecord | undefined {
+    ): ReportBacktraceDatabaseRecord | undefined {
         if (!this._enabled) {
             return undefined;
         }
 
-        this.prepareDatabase();
+        const sessionId = backtraceData.attributes?.['application.session'];
 
-        const record: BacktraceDatabaseRecord = {
-            count: 1,
+        const record: ReportBacktraceDatabaseRecord = {
+            type: 'report',
             data: backtraceData,
             timestamp: TimeHelper.now(),
             hash: '',
```

Comment:
i think we can also drop hash support if we're dropping count - by using hash we could count the same errors. Just in case, this doesn't need to happen in this pull request - this is information for the future

Reply:
Done